test/micro/org/openjdk/bench/java/lang/foreign/libQSortJNI.c

```diff
@@ -28,7 +28,7 @@
 #include "jlong.h"
 #include "JNICB.h"
 
-#ifdef _WIN64
+#if defined(_WIN64) || defined(_WIN32)
```

Comment:
`_WIN32` is defined on x64 as well [1], so I think this can just be `#ifdef _WIN32` to mean 'Windows'

[1]: https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170